script/Arcueid/Day 10/10_00_ARC10_7_2B.txt

```diff
@@ -960,6 +1001,7 @@ Her look invokes a seething sensation on my brain.
 [sha:c578347031e83e0236e652a02c65dd7fc6a88e97]{
 -- Page 821, Offset 26577.
 -- 充血する。失神しそうだ。
+// Is this clinical terminology? Or should we be a little meatier with it.
```

Comment:
the weblio definition sounds pretty clinical and I can't find anything on google (when i manage to avoid all the chinese) that makes this sound like anything but hyperemia